packages/stream-metadata/src/riverStreamRpcClient.ts

```diff
@@ -218,8 +220,7 @@ export async function getMediaStreamContent(
 	const result = await mediaContentFromStreamView(logger, sv, secret, iv)
 
 	// Cache the result
-	const concatenatedString = `${fullStreamId}${secretHex}${ivHex}`
-	contentCache[concatenatedString] = result
+	contentCache[cacheKey] = result
```

Comment:
@tak-hntlabs is this temporary until we introduce redis? we'll get out of memory errors - unless you have some custom ttl or cache invalidation logic running somewhere else. if we want to use local memory based caching (regardless of short term vs long term) we should look into [lru-cache](https://www.npmjs.com/package/lru-cache)